CONTRIBUTING.md

```diff
@@ -30,3 +30,41 @@ Use culturally neutral language:
 
   What you may consider well-known may be entirely obscure and distracting to people from different backgrounds.
 
+### Terminology
+
+To avoid confusion around all the things called Nix, always use the following terms with capitalization as given.
+
+Terms in brackets (`[  ]`) are optional.
+Terms in parentheses (`(  )`) are abbreviations.
+
+- Nix ecosystem
+
+  All of the below.
+
+  - Nix package manager
+
+    - `nix` [command [line interface]] (Nix CLI)
```

Comment:
Because of the many things that the Nix CLI is (and becoming): I would at least swap these two.

The _Nix package manager_ would be reflected in a subset of subcommands.

Maybe like so (actual names are no firm suggestion / only an examplification):

- `nix` ...
  - Implements: Nix Package Manager
  - Implements: Nix Language Evaluator
  - Implements: Nix REPL
  - Implements: Nix Flake Interface
  - ...